modules/windows-pod-placement.adoc

```diff
@@ -11,6 +11,8 @@ With multiple operating systems, and the ability to run multiple Windows OS vari
 
 For more information, see Microsoft's documentation on link:https://docs.microsoft.com/en-us/virtualization/windowscontainers/deploy-containers/update-containers#host-and-container-version-compatibility[Host and container version compatibility].
 
+Also, it is recommended that you set the `spec.os.name.windows` parameter in your workload pods. This field is used to authoritatively identify the pod operating system for validation and is used to enforce Windows-specific pod security context constraints (SCCs).
```

Comment:
Do we need to link to the [upstream doc](https://kubernetes.io/docs/concepts/workloads/pods/#pod-os) given this is a new option that is not widely known?

Reply:
Does our behavior match the Kubernetes behavior? 

Particularly: 
* In Kubernetes v1.26, the value you set for this field has no effect on scheduling of the pods. 

* The kubelet refuses to run a Pod where you have specified a Pod OS, if this isn't the same as the operating system for the node where that kubelet is running.

Reply:
Yes it does.

Reply:
So, in In Kubernetes v1.26/WMCO 8.0.0, the kubelet **DOES NOT** refuse to run a Pod where you have specified a Pod OS.......

Reply:
Sorry I was responding to your:

>Does our behavior match the Kubernetes behavior?

Yes it does.